hypha/apply/home/templates/apply_home/includes/apply_listing.html

```diff
@@ -1,23 +1,35 @@
 {% load i18n bleach_tags wagtailcore_tags markdown_tags heroicons %}
 
 {% if page.open_round %}
-    <div class="listing listing--not-a-link">
-        <div>
-            <h4 class="listing__title listing__title--link">
-                {% if page.next_deadline and page.show_deadline %}
-                    <p class="listing__deadline">
-                        {% heroicon_mini "calendar-days" class="inline me-1" aria_hidden=true %}
-                        <span>{% trans "Next deadline" %}: {{ page.next_deadline|date:"M j, Y" }}</span>
-                    </p>
-                {% endif %}
-                {{ page.title }}
-            </h4>
-            {% if page.description %}
-                <h6 class="listing__teaser">
-                    {{ page.description|markdown|bleach }}
-                </h6>
-            {% endif %}
-        </div>
-        <a class="listing__button" href="{% pageurl page %}">{% trans "Apply" %}</a>
+  <div class="flex justify-between items-center py-8">
```

Comment:
A "gap" would be good here I think. To get some space between the text and the button.

Reply:
Totally agree. I believe it got lost in the rebasing :D

Reply:
Added.